packages/redaction/README.md

```diff
@@ -23,3 +25,151 @@ const middleware = [
   }),
 ];
 ```
+
+## Utilties for redacting javascript objects
+
+To make it easier to redact large complex javascript types we provide a number of helpers.
+
+### asterixOut
+
+This ensures the redacted text is the same length and has the same line breaks as the original text,
```

Comment:
```suggestion
This ensures the redacted text is the same length and has the same whitespace as the original text,
```
I don't think we really guarantee that it has the same linebreaks, since if it's rendered in a non-monowidth font the linebreaks might end up in a different space.